src/app/rpmostree-builtin-status.c

```diff
@@ -422,10 +422,10 @@ status_generic (RPMOSTreeSysroot *sysroot_proxy,
         print_packages ("RemovedBasePackages", max_key_len,
                         origin_removed_base_packages, NULL);
 
-      /* let's be nice and only print requested - layered, rather than repeating
-       * the ones in layered twice */
-      if (origin_requested_packages)
-        print_packages ("RequestedPackages", max_key_len,
+      /* only print inactive layering requests in verbose mode */
+      if (origin_requested_packages && opt_verbose)
```

Comment:
Yeah, good point. Though it's kinda tricky only printing it when adding since the determination is made deep in the upgrader, which is designed to not care about the delta between the previous and current origin states.

I opted for simply making the upgrader always print inactive requests and inactive base removals. It makes the output a bit more chatty, though I think it makes sense to stick deductions like that in the upgrader output and keep the default status output more about what's currently active.